R/utils.R

```diff
@@ -0,0 +1,73 @@
+#' Creates `AVAL` from `AVALC` by Calling User Function
+#'
+#' Create `AVAL` from `AVALC` by calling a function provided by the user. If
+#' calling the function fails, the error is catched and a helpful error message
```

Comment:
catched -> caught

Reply:
Strange that the spelling check passed. If I run it locally, the error is found.

@cicdguy , could you have a look? There is also a warning in the log of the spellcheck. I am not sure if it is expected.

Reply:
The warning can be ignored.
Apparently `catched` is a word that is defined in the Merriam Webster dictionary: https://www.merriam-webster.com/dictionary/catched
I would have never catched that.

Reply:
Interesting!
Thanks, @cicdguy